deploy/concierge/helpers.lib.yaml

```diff
@@ -45,3 +45,62 @@ _: #@ template.replace(data.values.custom_labels)
 #@   end
 #@   return log_level
 #@ end
+
```

Comment:
I wish we could make this a `.star` file so that most of the file doesn't have to be commented out.